.gitignore

```diff
@@ -20,3 +20,4 @@ cabal.project.local
 cabal.project.local~
 .HTF/
 .ghc.environment.*
+.stylish-haskell.yaml
```

Comment:
This file is used by the formatter that I'm using. I'm adding it as `.gitignore` as I'm not sure if I'm allowed to add the file.